ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organization names that use Quarkus based on public information available in blog posts and videos. 
```

Comment:
```suggestion
This document lists the organizations that use Quarkus based on public information available in blog posts and videos. 
```

Reply:
This change feels more natural to me, but I'm not an English native speaker so maybe the original version is better.